src/Components/Facility/Consultations/components/SharedTable.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { formatDate, formatTime } from "../../../../Utils/utils";
+import { classNames } from "../../../../Utils/utils";
+
+interface SharedSectionTableProps {
+  data: Record<string, any>;
+  rows: Array<{ title?: string; field?: string; subField?: boolean }>;
+  choices?: Record<string, Record<number | string, string>>;
+  translateKey: string;
+  t: (key: string) => string;
+}
+
+const SharedSectionTable: React.FC<SharedSectionTableProps> = ({
+  data,
+  rows,
+  choices = {},
+  translateKey,
+  t,
+}) => {
```

Comment:
1. Rename this to be `LogUpdateAnalayseTable` since it's built for it.
2. use `useTranslations` instead of passing it as a prop.
3. i18n suffix `LOG_UPDATE_FIELD_LABEL__` is not going to change, so skip passing it as a prop.

```suggestion
const LogUpdateAnalayseTable: React.FC<SharedSectionTableProps> = ({
  data,
  rows,
  choices = {},
}) => {
```